lib/util/constants.dart

```diff
@@ -206,7 +204,7 @@ const AUTONOMY_DOCUMENT_PREFIX = 'https://github.com/bitmark-inc';
 const AUTONOMY_RAW_DOCUMENT_PREFIX =
     'https://raw.githubusercontent.com/bitmark-inc';
 
-const MARKDOWN_EXT = '.md';
+const markdownExt = '.md';
```

Comment:
use UPPER_CASE for constant

Reply:
dart convention is to use camel, checkout dart analysis